server/commands.lua

```diff
@@ -1,109 +1,36 @@
-QBCore.Commands = {}
-QBCore.Commands.List = {}
-QBCore.Commands.IgnoreList = { -- Ignore old perm levels while keeping backwards compatibility
-    god = true, -- We don't need to create an ace because god is allowed all commands
-    user = true -- We don't need to create an ace because builtin.everyone
-}
-
-CreateThread(function() -- Add ace to node for perm checking
-    local permissions = QBConfig.Server.Permissions
-    for i=1, #permissions do
-        local permission = permissions[i]
-        ExecuteCommand(('add_ace qbox.%s %s allow'):format(permission, permission))
-    end
-end)
-
--- Register & Refresh Commands
-
-function QBCore.Commands.Add(name, help, arguments, argsrequired, callback, permission, ...)
```

Comment:
Why not deprecate the function rather than remove it?

Reply:
It's either deprecating or removing, and at this point idk which one to do honestly

Reply:
Removing will be a breaking change. We should deprecate.